src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -805,7 +805,7 @@ public SyntaxNode Attribute(AttributeData attribute)
 
             return Attribute(
                 name: this.TypeExpression(attribute.AttributeClass),
-                attributeArguments: args.Count > 0 ? args : null);
+                attributeArguments: args.Length > 0 ? args.AsEnumerable() : null);
```

Comment:
:bulb: `!args.IsEmpty` (definitely works for `ImmutableArray<T>` but I can't tell from this code if we're looking at `ImmutableArray<T>` or `ImmutableList<T>`)